core/common/src/main/java/alluxio/underfs/ObjectUnderFileSystem.java

```diff
@@ -633,18 +634,17 @@ public boolean mkdirs(String path, MkdirsOptions options) throws IOException {
 
   @Override
   public InputStream open(String path, OpenOptions options) throws IOException {
-    return openObject(stripPrefixIfPresent(path), options);
+    return openObject(stripPrefixIfPresent(path), options, null);
```

Comment:
instead of null can we use a retrypolicy which only attempts once?

Reply:
Changed, now get `new CountingRetry(1)` when using `open()`.Thanks